aws-codeguruprofiler-profilinggroup/pom.xml

```diff
@@ -74,9 +74,19 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>codeguruprofiler</artifactId>
-            <version>2.10.63</version>
         </dependency>
     </dependencies>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>bom</artifactId>
+                <version>2.13.16</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
Would be nice to see this change in a separate commit and with an explanation of what it does, so that it would be clear and we could refer to it in the future.

Reply:
This was later extracted by @gimki as part of [PR #11](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeguru-profiler/pull/11).